src/election.ts

```diff
@@ -197,6 +219,26 @@ export const getBallotStyle = ({
 }): BallotStyle | undefined =>
   election.ballotStyles.find((bs) => bs.id === ballotStyleId)
 
+/**
+ * Retrieve a contest from a set of contests based on ID
+ * special-cases Ms Either Neither contests
+ */
+const findContest = ({
```

Comment:
```suggestion
export const findContest = ({
```

Maybe useful elsewhere?